Quadriga/src/main/webapp/WEB-INF/views/projectCollections/addConceptCollection.jsp

```diff
@@ -0,0 +1,59 @@
+<!--  
```

Comment:
hm, ok, so you copy pasted it? Why do you copy that file? It doesn't seem to have to do anything with a project blog. Also, why is the folder called "projectCollections"? Aren't you working on a project blog?